src/api/java/baritone/api/utils/ExampleBaritoneControl.java

```diff
@@ -662,10 +719,43 @@ public boolean isInGoal(int x, int y, int z) {
         }
         if (msg.equals("damn")) {
             logDirect("daniel");
+            return true;
```

Comment:
why did you mark this as resolved it was not even removed 🤔

Reply:
Without `return true;` or with `return false;` the command will return: 
`[Baritone] daniel`
`[Baritone] Invalid command`

With `return true;` the command will return: 
`[Baritone] daniel`

Reply:
For the last time, this is **intended behavior**, don't add this `return true;`. This will not be merged until the `return true;` is removed.